mantidproject/mantid
# Hide Engineering Diffraction 1 and Rename Eng Diff 2

**Description of work.**
Hides the engineering diffraction 1 interface since it isn't used.
Renames eng diff 1 and 2 to `Engineering Diffraction (Old)` and `Engineering Diffraction` respectively. 

**To test:**
1. Check that Diffraction -> Engineering Diffraction appears in the list of workspaces in both plot and workbench.
2. Check that the new GUI opens, not the old one.
3. Check documentation still builds and makes sense. 

Fixes #28091 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
